admin-base/core/src/main/java/com/peregrine/admin/resource/AdminResourceHandlerService.java

```diff
@@ -354,6 +355,26 @@ public Resource createObjectDefinition(ResourceResolver resourceResolver, String
         }
     }
 
+    private void addFile(Node parent, String name, String data) throws RepositoryException {
+        Node file = parent.addNode(name, "nt:file");
+        Node fileNode = file.addNode("jcr:content", "nt:resource");
+        fileNode.setProperty("jcr:mimeType", "application/json");
+        fileNode.setProperty("jcr:encoding", "UTF-8");
+        fileNode.setProperty("jcr:data", new StringBufferInputStream(data));
```

Comment:
I would use constants here, but since it's just an extraction, I can let it slip. Though I encourage to use `JcrConstants`, if you want me to - I can replace them for you.

Reply:
agree, this should be changed - @reggie7 would you mind changing this? thank you

Reply:
done